plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -89,6 +92,15 @@ const CatalogPage: FC<{}> = () => {
         hidden: location ? location?.type !== 'github' : true,
       };
     },
+    (rowData: Component) => {
+      return {
+        icon: starredEntities.has(rowData.metadata.name) ? Star : StarOutline,
+        toolTip: `${
+          starredEntities.has(rowData.metadata.name) ? 'Unstar' : 'Star'
```

Comment:
is this the nomenclature? should it be `Add to favorites` and `Remove from favorites` instead? @katz95 @stefanalund

Reply:
Yes, what he said.